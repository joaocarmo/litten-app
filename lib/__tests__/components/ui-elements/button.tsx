import TestRenderer from 'react-test-renderer'
import ThemeProvider from '@components/theme/provider'
import { UIButton } from '@ui-elements'

describe('Snapshot test for the "UIButton" component', () => {
  it('renders correctly', () => {
    const element = TestRenderer.create(
      <ThemeProvider>
        <UIButton>Some text here</UIButton>
      </ThemeProvider>,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })
})
