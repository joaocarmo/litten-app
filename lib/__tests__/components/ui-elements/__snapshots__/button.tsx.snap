// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the "UIButton" component renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  primary={true}
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#7239f2",
        "borderRadius": 24,
        "justifyContent": "center",
        "minHeight": 44,
      },
      {
        "width": 577.5,
      },
      undefined,
      undefined,
    ]
  }
>
  <Text
    style={
      {
        "color": "#ffffff",
        "fontSize": 15.75,
        "fontWeight": "700",
      }
    }
  >
    Some text here
  </Text>
</View>
`;
